src/decode/lzma.rs

```diff
@@ -38,14 +37,39 @@ enum ProcessingStatus {
     Finished,
 }
 
-pub struct LzmaParams {
+#[derive(Copy, Clone)]
+/// LZMA 'lclppb' decompression properties.
+pub struct LzmaProperties {
     // most lc significant bits of previous byte are part of the literal context
```

Comment:
Nit: This explanation can probably be merged with the following line, i.e.:

```
/// The number of literal context bits.
///
/// The most `lc` significant bits of the previous byte are part of the literal context.
```

Reply:
https://github.com/gendx/lzma-rs/pull/73/commits/5716a6bca5872c33ff806ccfd6a87b6ea90e31b9#diff-768757151baad4fb0605aaeee5210dc343977125d64711ebcf5a0e755b2f23c0R45